### ISSUE TITLE
Developer Accessibility Issues

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Developer#32]
I ran Compliance Sheriff against the new developer.dol.gov pages (50), the accessibility issues had to do with two main parts: the search field and TABLEs.

The search fields were missing a LABEL and text that can be placed visually off-screen and the ALT text for the magnifying glass, which should be "Search".

The TABLEs in the dataset pages need to SUMMARY statements, CAPTIONs, and the TH connected to the TD cells by scope="col".

I can sit with you Neil on Monday or Tuesday to either help you make these changes or possible do so of them for you.

The WebAIM toolbar found an issue with the missing anchor for the skip link. We should also make it visible when we tab to the link so people with mobility issues will see it then use it.

There are most likely other issues that need to be corrected, but this is a start.

**Comment by jfc3-dol** [comment]
Had the tech team correct the missing SEARCH LABEL, along with adding the ALT text of "Search" for the magnifying glass button/icon.

The first pass of fixing the dataset pages I added SUMMARY statements, CAPTIONs, TH (for TABLE headings), and scope="col" too. Once these changes were made and checked by QA, we moved the changes to PROD/GitHub.

Next we created a mobile friendly version of each of the TABLEs, which meant having both the TABLE and mobile version on the same page. I accomplished this by creating two different CLASSes that would show one and hide the other depending on the screen size. Doing this allows a person to look at the information that is contained in each TABLE on their mobile device without having to scroll left and right.

We also made other changes detailed in issue #58 to allow keyboard users to correct internal page links with Safari and Chrome.

We can close this issue since we pushed these changes to PROD/GitHub a while ago.

**Comment by adams-lisa** [comment]
QA validation complete.  Closed
